test/tester.go

```diff
@@ -436,8 +436,10 @@ func (at *AccountsTester) TrackDownload(skylink string, bytes int64) (int, error
 }
 
 // TrackUpload performs a `POST /track/upload/:skylink` request.
-func (at *AccountsTester) TrackUpload(skylink string) (int, error) {
-	r, _, err := at.request(http.MethodPost, "/track/upload/"+skylink, nil, nil, nil)
+func (at *AccountsTester) TrackUpload(skylink string, ip string) (int, error) {
+	form := url.Values{}
+	form.Set("ip", ip)
+	r, _, err := at.request(http.MethodPost, "/track/upload/"+skylink, form, nil, nil)
```

Comment:
I might be confusing things but didn't we introduce `at.post` in another PR at some point?

Reply:
I think I used to have those before but I decided that i don't need that many layers of abstraction. If they're not to be used outside of tester's implementation then it's more overhead than help.

Also, I believe Pj is using that approach in one of the abuse services.